app/grandchallenge/archives/models.py

```diff
@@ -33,7 +34,12 @@
 from grandchallenge.subdomains.utils import reverse
 
 
-class Archive(UUIDModel, TitleSlugDescriptionModel, ViewContentMixin):
+class Archive(
+    UUIDModel,
+    TitleSlugDescriptionModel,
+    ViewContentMixin,
```

Comment:
You will need to merge from main as `ViewContentMixin` -> `HangingProtocolMixin`.